### PR TITLE
Add requirements.txt for virtual environments

### PR DESCRIPTION
**Pull request by BlossomiShymae**
This is a nice thingie to have for people who use python virtual environments, so they can install from requirements instead of figuring out that a package is missing.

3765_EM_HAPPY_BIRTHDAY_TO_AHRI_Inventory_100x100 blossoms

### CODE REVIEW

**Comment by DarkIntaqt** [comment]
That's a nice addition. Thanks a lot!

If you want, you can edit https://github.com/DarkIntaqt/lolassethelper/blob/master/SETUP.md too and add an installation step on how to install the requirements

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+aiohttp==3.8.4
```
Comment (DarkIntaqt) [inline requirements.txt:1]: I think a missing requirement is asycio

**Comment by BlossomiShymae** [comment]
Okie dokie! I managed to add an installation step covering it. :3

File: requirements.txt (hunk 1 above)
Comment (DarkIntaqt) [reply requirements.txt:1]: Ignore this, asnycio is built-in; My bad

**Review by DarkIntaqt — APPROVED** [review]
Looks good

**Pull request merged**